source/operations/concepts/availability-and-resiliency.rst

```diff
@@ -0,0 +1,157 @@
+      :align: center
+      :alt: Diagram of erasure coded object partitioned into 12 (twelve) data shards and 4 (four) parity shards
+
+      This small one-node deployment has 16 drives in one erasure set.
+      Assuming default :ref:`parity <minio-ec-parity>` of ``EC:4``, MinIO shards the object into 4 (four) parity shards and 12 (twelve) data shards.
+      MinIO distributes these shards evenly across each drive in the erasure set.
+
+MinIO uses a deterministic algorithm to select the erasure set for a given object.
+   For each unique object namespace ``BUCKET/PREFIX/[PREFIX/...]/OBJECT.EXTENSION``, MinIO always selects the same erasure set for read/write operations.
+   This includes all :ref:`versions <minio-bucket-versioning>` of that same object.
+
+   .. figure:: /images/availability/availability-erasure-set-selection.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of erasure set selection based on object namespace
+
+      MinIO calculates the destination erasure set using the full object namespace.
+
+MinIO requires :ref:`read and write quorum <minio-read-quorum>` to perform read and write operations against an erasure set.
+   The quorum depends on the configured parity for the deployment.
+   Read quorum always equals the configured parity, such that MinIO can perform read operations against any erasure set that has not lost more drives than parity.
+
+   .. figure:: /images/availability/availability-erasure-sharding-degraded.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of degraded erasure set, where two parity shards replace two data shards
+
+      This node has two failed drives.
+      MinIO uses parity shards to replace the lost data shards automatically and serves the reconstructed object to the requesting client.
+
+   With the default parity of ``EC:4``, the deployment can tolerate the loss of 4 (four) drives per erasure set and still serve read operations.
+
+Write quorum depends on the configured parity and the size of the erasure set.
+   If parity is less than 1/2 (half) the number of erasure set drives, write quorum equals parity and functions similarly to read quorum.
+   MinIO automatically "upgrades" the parity of objects written to a degraded erasure set to ensure that object can meet the same :abbr:`SLA (Service Level Agreement)` as objects in healthy erasure sets.
+
+   .. figure:: /images/availability/availability-erasure-sharding-degraded-write.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of degraded erasure set, where two drives have failed
+
+      This node has two failed drives.
+      MinIO writes the object with an upgraded parity of ``EC:6`` to ensure this object meets the same SLA as other objects.
+
+   With the default parity of ``EC:4``, the deployment can tolerate the loss of 4 drives per erasure set and still serve write operations.
+   MinIO can perform "parity upgrade" up to /2 the drives in the erasure set.
```

Comment:
Will drop this line. I think we have to talk about it though, since it does affect how objects work after quorum goes back to normal.  Did a tweak or two.